### PR TITLE
Removed master feed and fixed copying Nuget.config

### PR DESCRIPTION
**Pull request by suhasj**
@pranavkm 

- Removed myget/master feed
- Added check to copy Nuget.config only if present

### CODE REVIEW

File: build-template/NuGet.master.config [hunk 1]
```diff
@@ -2,6 +2,5 @@
 <configuration>
   <packageSources>
     <add key="NuGet" value="https://nuget.org/api/v2/" />
-    <add key="AspNetMaster" value="https://www.myget.org/F/aspnetmaster/api/v2" />
```
Comment (pranavkm) [inline build-template/NuGet.master.config]: Oops. We checked with @danroth27 and it sounded like we were pushing all our packages to `nuget.org`. We dorely on this for bringing in packages like `KoreBuild` that don't end up on NuGet.org.

@suhasj we might need to do another update to the master branches with this feed in it.

Comment (Eilon) [inline build-template/NuGet.master.config]: Yeah we definitely don't push *everything* to NuGet.org. Master will always be needed.

Comment (suhasj) [inline build-template/NuGet.master.config]: ah let me sync offline

**Comment by suhasj** [comment]
Updated repos and reverted the changes in this PR

**Comment by suhasj** [comment]
Ping @pranavkm

**Comment by davidfowl** [comment]
What does this do?

**Comment by suhasj** [comment]
super trivial change actually. Couple of repros like Coherence-Signed dont have a Nuget.config and hence this checks if that is the case then we don't replace the Nuget.config before pushing. It throws an error if this not there.

**Comment by suhasj** [comment]
closing this since will be adding more features

**Pull request closed without merging**